Capstone/Autocorrect/AutocorrectFunctions.h

```diff
@@ -4,17 +4,14 @@
 //
 //  Created by Abel Kelbessa on 7/14/22.
 //
-
 #import <Foundation/Foundation.h>
 
 NS_ASSUME_NONNULL_BEGIN
-
 @interface AutocorrectFunctions : NSObject
 + (instancetype)shared;
 @property (strong, nonatomic) NSMutableDictionary *keyCoordinates;
-- (void)createCoordinates;
-- (double)hammingDistance:(NSString *)string1 :(NSString *)string2;
-- (double)hammingDistance2:(NSString *)character1 :(NSString *)character2;
+- (NSString *)findCorrectWord:(NSString *)searchWord :(NSArray *)colleges;
+- (double)hammingDistanceCalculator:(NSString *)string1 :(NSString *)string2;
```

Comment:
lets try and make this variable a bit more descriptive than string2!

Reply:
Done!